docs/components/DataList/DataList.stories.mdx

```diff
@@ -23,3 +25,201 @@ item.
 If you need to add more information, consider using the
 [DataTable](?path=/story/components-lists-and-tables-datatable-docs--page)
 component instead.
+
+## Implementation guidelines
+
+Since DataList is a large component, we use a mixture of compound components and
+props to separate each part of the component.
+
+## Main wrapper
```

Comment:
```suggestion
### Main wrapper
```

Reply:
I was actually doing this so it shows up on the TOC haha